src/jmc/compile/datapack.py

```diff
@@ -564,3 +564,10 @@ def __repr__(self) -> str:
     loads = {dumps(self.loads, indent=4)}
     tick = {dumps(self.ticks, indent=4)}
 )"""
+
+def func_with_namespace(func: str, datapack: DataPack) -> str:
```

Comment:
- Rename `func_with_namespace` with more intuitive name
- Please use docs string to shortly exxplain what it does in the correct format
- If you are absolutely sure it should be in datapack, put it inside the class instead of a floating function, if not then try to move it where it should. If you can't figure out how to fix the circular import then you can ask me.

Reply:
1) `format_func_namespace`?
2) Sure, I'll add that
3) Yeah, I just need your help fixing the circular import. I originally wanted to make it part of `utils.py` or `lexer_func_content.py` but both of those ran into the problem